protocol/messenger_peersyncing.go

```diff
@@ -118,12 +118,30 @@ func (m *Messenger) sendDatasyncOffers() error {
 		return nil
 	}
 
-	communities, err := m.communitiesManager.Joined()
+	err = m.sendDatasyncOffersForCommunities()
 	if err != nil {
 		return err
 	}
 
-	for _, community := range communities {
+	err = m.sendDatasyncOffersForChats()
+	if err != nil {
+		return err
+	}
+
+	// Check all the group ids that need to be on offer
```

Comment:
Is this comment misplaced?

Reply:
This comment was actually there before (https://github.com/status-im/status-go/blob/develop/protocol/messenger_peersyncing.go#L169), it's question to @cammellos who was writing this code.

Reply:
oh sorry, it was implementation comment, can be removed